modules/auxiliary/admin/mssql/mssql_escalate_execute_as.rb

```diff
@@ -24,6 +24,7 @@ def initialize(info = {})
   def run
     # Check connection and issue initial query
     print_status("Attempting to connect to the database server at #{rhost}:#{rport} as #{datastore['USERNAME']}...")
+
```

Comment:
I'm guessing this left over from a rebase; we should be good to revert this change :+1: